.bazelrc

```diff
@@ -6,6 +6,7 @@ build --strategy=TypeScriptCompile=sandboxed,local
 
 # Compatibility flags to avoid regressions.
 build --incompatible_disallow_empty_glob
+query --incompatible_disallow_empty_glob
```

Comment:
does just putting this in `common` work?

Reply:
thanks, didn't see that.